src/PrestaShopBundle/Form/Admin/Sell/Product/ProductPriceType.php

```diff
@@ -58,6 +58,11 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             ->add('tax_rule_group_id', ChoiceType::class, [
                 'choices' => $this->taxRuleGroupChoiceProvider->getChoices(),
                 'required' => false,
+                'placeholder' => false,
+                'attr' => [
+                    'data-toggle' => 'select2',
+                    'data-minimumResultsForSearch' => '7',
```

Comment:
Must be a const, same for divisor above

Reply:
Agree, where do you think we should define it? This value is used in multiple forms.

Reply:
Could be in its own class, or in an interface.
Ping @matks senseï!

Reply:
What about a dedicated Settings class for produts, like https://github.com/PrestaShop/PrestaShop/blob/develop/src/Core/Domain/Category/SeoSettings.php ?

Example `PrestaShop\PrestaShop\Core\Domain\Product\ProductSettings` or `Domain\BackOffice\BackOfficeSettings` (it's definitely related to how the BO behaves) ?

Reply:
For some reason I don't feel like this constant should be part of Domain. :/ It should be closer to Controller layer imo. :thinking:

Reply:
Can be in Controller too indeed. It depends whether we see it as a configuration setting of the backoffice interface, just like "which icon do we use" or "which label do we use" or as a configuration setting of the interface as a domain. In this last case, this value could be used either for the BO interface or in a API as well for example.

Controller is the shortest and simplest way => let's go for it